source/common/less/theme-berlin/dark.less

```diff
@@ -112,6 +112,16 @@ body.dark {
       background-color: #333;
     }
 
+    .code-block-line-open {
```

Comment:
I suggest we call the classes `code-block-first-line` and `code-block-last-line`. The open/close metaphor works better for inline elements. At least that's my impression here.